src/public_parameters/instance.rs

```diff
@@ -115,20 +130,24 @@ impl Metadata {
     }
 }
 
-impl<F: CurveCycleEquipped, C: Coprocessor<F>> Instance<F, C>
+impl<
+        'a,
+        F: CurveCycleEquipped,
+        C: Coprocessor<F>,
+        M: MultiFrameTrait<'a, F, C> + SuperStepCircuit<F> + NonUniformCircuit<G1<F>, G2<F>, M, C2<F>>,
```

Comment:
Thanks for going through this. I'll simplify `MultiFrameTrait` once LEM merges. 🎖️

Reply:
Thanks @gabriel-barrett!